tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -232,6 +235,38 @@ Object.Pipeline {
 			}
 		}
 	}
+
+	passthrough-capture.2 {
+		index	7
+
+		Object.Widget.copier.1 {
+			stream_name	'Echo reference'
```

Comment:
Can you make this string "Echo reference" a variable defined in the Define {} block so you can use it here and in the pcm caps object below?

Reply:
Ack